Package.swift

```diff
@@ -230,13 +230,12 @@ let package = Package(
       ]
     ),
 
-    .executableTarget(
-      name: "Java2Swift",
+    .target(
+      name: "Java2SwiftLib",
```

Comment:
Not sure about the “lib” suffix tbh, just java2swift is fine

Reply:
I'd have to rename the executable target to drop the Lib suffix